### PR TITLE
Rename register templates to be less redundant

### PR DESCRIPTION
**Pull request by rogeruiz**
When the URL path was `/states/` having the prefix `register-` made more sense. I missed this step in the #63.

### CODE REVIEW

**Comment by maya** [comment]
@rogeruiz I think we had the `register-` prefix so you didn't get confused with the `single.html` template, since those were the four types of registration template. Is there any way to add clarity around that or move the `single.html` template somewhere else? Or do you think it's not a big deal?

**Comment by rogeruiz** [comment]
@maya Hmm... I'd say the folder is now the new `register/` prefix. The `single.html` is what is generating the partials in there using `{{ Render /* ... */ }}`. We could add a `register/partials/`or `register/types` folder in there. But the `register/single.html` is a Hugo-thing which renders the header and footer of the `register/**` pages and calls the other templates in that directory.

File: layouts/register/single.html [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-  {{ $registration_type := .Params.registration_type }}
-
-  {{ $registration_template := printf "register-%s" $registration_type }}
-
   <body class="page-register-{{ .Params.registration_type }}">
     {{ partial "header.html" . }}
     <div id="main-content" role="main">
 
-      {{ .Render $registration_template }}
+      {{ .Render .Params.registration_type }}
```
Comment (rogeruiz) [inline layouts/register/single.html]: We could update this with a path like `{{ .Render printf "`./types/" .Params.registration_type }}` here.

**Comment by maya** [comment]
@rogeruiz what would go in those folders: the single.html or the four templates?

File: layouts/register/single.html [hunk 2]
```diff
@@ -1,14 +1,10 @@
 {{ partial "head.html" . }}
```
Comment (rogeruiz) [inline layouts/register/single.html]: We could add a `{{ /* Hugo Comment */ }}` here that better describes what this is with documentation source.

**Comment by rogeruiz** [comment]
The four templates would go in there.

```
.
├── [ 204]  types
│   ├── [ 373]  online.html
│   ├── [ 330]  not-needed.html
│   ├── [ 389]  in-person.html
│   └── [1.6K]  by-mail.html
└── [ 517]  single.html

1 directory, 5 files
(END)
```

**Comment by maya** [comment]
@rogeruiz that works for me!

**Comment by rogeruiz** [comment]
I updated the PR with the scoped directory name `types/` @maya

**Comment by maya** [comment]
LGTM 🎸

**Pull request merged**